DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -178,6 +184,43 @@ public IActionResult AddRegistrationPromptSummaryPost()
             return RedirectToAction("Error", "LearningSolutions");
         }
 
+        [HttpGet]
+        [Route("{promptNumber}/Remove")]
+        public IActionResult RemoveRegistrationPrompt(int promptNumber)
+        {
+            var delegateWithAnswerCount =
+                userDataService.GetDelegateCountWithAnswerForPrompt(User.GetCentreId(), promptNumber);
+
+            if (delegateWithAnswerCount == 0)
+            {
+                return TryRemoveRegistrationPrompt(promptNumber);
+            }
+
+            var customPrompts = customPromptsService.GetCustomPromptsForCentreByCentreId(User.GetCentreId());
+            var promptName = customPrompts.CustomPrompts.Single(c => c.CustomPromptNumber == promptNumber)
+                .CustomPromptText;

```

Comment:
I think this is used in a few place now - do you think it's worth creating a method to get prompt based on prompt number/id for centre?

Reply:
Note here, this was the only place where it was being done like this. There is another place where a similar thing was being done, but with different input data. Do you want me to write another separate method to handle that?